src/ontology/cl-edit.owl

```diff
@@ -32242,6 +32244,15 @@ SubClassOf(obo:CL_4042016 obo:CL_0000313)
 SubClassOf(obo:CL_4042016 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005385))
 SubClassOf(obo:CL_4042016 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0011148))
 
+# Class: obo:CL_4047006 (fetal vein endothelial cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34497389") obo:IAO_0000115 obo:CL_4047006 "An endothelial cell that lines the blood vessels in the fetal circulatory system.")
+AnnotationAssertion(terms:contributor obo:CL_4047006 <https://orcid.org/0009-0005-7919-4905>)
+AnnotationAssertion(terms:date obo:CL_4047006 "2024-08-15T13:05:25Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4047006 "fetal vein endothelial cell")
+EquivalentClasses(obo:CL_4047006 ObjectIntersectionOf(obo:CL_0000115 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001638)))
```

Comment:
This equivalent class is used for 'vein endothelial cell', this is the reason the PR is failing.  You can either separate the axioms or add 'part of' some 'late embryo' . I'm not a big fan of this last option.